src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -322,6 +323,8 @@ const RecipientTable = ({
               onClick={() => {
                 setKey(record.recipient_id);
                 setEditing(true);
+                setRecipientToEdit(record);
+                console.log('new state recipient', key);
```

Comment:
Another console.log. Maybe we should ask Ash about this because there are plenty of them currently in production, so maybe it is fine.

Reply:
Good catch! I'll ask them for future PRs.